cdk-infra/utils/buckets.ts

```diff
@@ -0,0 +1,12 @@
+export const docsBucketNames = [
+  'docs-mongodb-org',
+  'docs-atlas',
+  'docs-atlas-osb',
```

Comment:
I moved this to a separate file to cut back on the size of the `buckets-construct.ts` file, but looking at it now, I could probably just move it to the `buckets-construct.ts` since this array isn't that big. Thoughts?

Reply:
I think it's fine to keep as-is for the sake of organization, but no strong preferences